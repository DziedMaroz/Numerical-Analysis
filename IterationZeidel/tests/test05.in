4
2 1 4 3 10
5 3 5 3 16
5 1 8 6 20
7 3 5 3 18
1 1 1 1 
0.005

