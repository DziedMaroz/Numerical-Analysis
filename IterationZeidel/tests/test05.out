RUNNING ON TEST: tests/test05.in
SIMPLE ITERATION:
K = 662
  inf   inf  -nan   inf 
ZEIDEL:
K = 40
-1.008 -0.947 -1.008 -1.063 

